internal/proto/buf.yaml

```diff
@@ -1,4 +1,8 @@
 version: v1
+name: buf.build/hashicorp/boundary
+deps:
+  - buf.build/googleapis/googleapis
+  - buf.build/grpc-ecosystem/grpc-gateway
```

Comment:
These are modules we depend on, one is maintained by Buf (googleapis) and one is the official grpc-gateway module (grpc-gateway).